delfin/task_manager/scheduler/schedule_manager.py

```diff
@@ -76,6 +79,15 @@ def schedule_boot_jobs(self):
                 LOG.error("Failed to initialize periodic tasks, reason: %s.",
                           six.text_type(e))
                 raise e
+        job_generator = service. \
```

Comment:
IMO, This service is not required. We can put the JobGenerator under metrics_manager itself

Reply:
This service should be only on leader node, so I create a new service here